clientconn.go

```diff
@@ -973,7 +968,11 @@ func (ac *addrConn) resetTransport() {
 		backoffFor := ac.dopts.bs.Backoff(ac.backoffIdx)
 
 		// This will be the duration that dial gets to finish.
-		dialDuration := getMinConnectTimeout()
+		dialDuration := minConnectTimeout
+		if ac.dopts.getMinConnectTimeoutFunc != nil {
```

Comment:
Nit: you can probably leave out "get" and "Func".

Reply:
Done